internal/resolution/manifest/maven.go

```diff
@@ -24,8 +24,8 @@ const (
 )
 
 type MavenManifestSpecific struct {
-	Properties      []PropertyWithOrigin
-	OriginalImports []resolve.RequirementVersion
+	Properties           []PropertyWithOrigin
+	OriginalRequirements []resolve.RequirementVersion
```

Comment:
Should we add a  comment or name this variable a bit better to capture the fact that these are requirements with properties only?

RequirementsWithProperties?

Reply:
done!